fiftyone/utils/openworld_sam.py

```
@@ -0,0 +1,432 @@
+            arr = img.detach().permute(1, 2, 0).cpu().numpy()
+            if arr.dtype != np.uint8:
+                arr = (arr * 255).clip(0, 255).astype(np.uint8)
+            return OpenWorldSAMModel._ensure_rgb(arr)
+        if isinstance(img, np.ndarray):
+            if img.dtype != np.uint8:
+                img = (img * 255).clip(0, 255).astype(np.uint8)
+            return OpenWorldSAMModel._ensure_rgb(img)
+        return OpenWorldSAMModel._ensure_rgb(np.asarray(img).astype(np.uint8))
+
+    @staticmethod
+    def _ensure_rgb(arr):
```

Ruff (0.15.18)

[warning] Missing return type annotation for staticmethod `_ensure_rgb`

(ANN205)